### ISSUE TITLE
Focus State for Page

### ISSUE THREAD

**Issue by mgifford** [GSA/openacr#332]
It is normal to have the focus be at the top of the page when it loads. This helps screen reader users orient themselves. 

When you load the site for the first time, either on the Overview page or subsequent pages, this works as expected:
https://gsa.github.io/openacr-editor/
https://gsa.github.io/openacr-editor/chapter/functional_performance_criteria

After navigating within the site, the focus seems to be at the top of the main content of the site, but below the header and navigation.

<img width="710" alt="Screen Shot with Valid status highlighted." src="https://user-images.githubusercontent.com/116832/153895604-38ed96f8-9917-47c6-94b2-facfe62f1c84.png">

There are arguably usability issues that could be argued. Ultimately it would be beneficial to have input from screen reader users if there is a preference.

**Comment by dmundra** [comment]
Agreed. On further inspection this might be less a bug and a feature of using `<nav>` with at least Chrome.
